math/minuit2/src/MnHesse.cxx

```diff
@@ -95,7 +95,7 @@ operator()(const MnFcn &mfcn, const MinimumState &st, const MnUserTransformation
 {
    // internal interface from MinimumState and MnUserTransformation
    // Function who does the real Hessian calculations
-   MnPrint print("MnHesse");
+   MnPrint print("MnHesse", MnPrint::GlobalLevel());
```

Comment:
I do not understand the need for this, since the constructor of MnPrint has a default level=MnPrint::GlobalLevel(). 
See MnPrint.h line 98. 
This PR looks is not changing anything

Reply:
Hi Lorenzo, thanks for the review! Unfortunatley, I can't really remember what I was trying to do at the time. Feel free to close the PR for now, if the issue pops up again at any point in the future, I'll revive it.

Reply:
OK, thanks for your reply. Let's do this then